backend/.golangci.yml

```diff
@@ -3,8 +3,6 @@ linters:
   enable:
     - asciicheck
     - bodyclose
-    - deadcode
```

Comment:
Were they removed because they don't work anymore or just that they are deprecated? Maybe it can be that we continue using them a bit longer if they still work.

I wonder if there are alternatives for the ones removed?

Reply:
Many of them were deprecated and suggested alternatives which I have added in the PR. For example deadcode is replaced by `unused`